packages/server-ct/src/project-ct.ts

```diff
@@ -1,19 +1,15 @@
 import Debug from 'debug'
-import config from '@packages/server/lib/config'
-import plugins from '@packages/server/lib/plugins'
-import devServer from '@packages/server/lib/plugins/dev-server'
 import { Cfg, ProjectBase } from '@packages/server/lib/project-base'
-import settings from '@packages/server/lib/util/settings'
-import specsUtil from '@packages/server/lib/util/specs'
 import { ServerCt } from './server-ct'
-import { SpecsStore } from './specs-store'
+import { SocketCt } from './socket-ct'
+import { createRoutes } from './routes-ct'
 
 export * from '@packages/server/lib/project-base'
 
 const debug = Debug('cypress:server-ct:project')
 
 export class ProjectCt extends ProjectBase<ServerCt> {
-  get projectType () {
+  get projectType (): 'ct' {
```

Comment:
nit: It's funny how we have this component in some places and as e2e at others.
A quick pass of consistency would not hurt.

Reply:
Yep, agreed. Happy to do this in a separate PR - I think that'll be easier to review and less likely to introduce a bug. We should have a singled shared type we use throughout the code base.